tests/test_curated_papers.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+"""Test for checking the integrity of the curated_papers TSV file."""
+
+import csv
+import unittest
+from datetime import datetime
+
+import bioregistry
+from bioregistry.constants import CURATED_PAPERS_PATH, ORCID_PATTERN
+from bioregistry.curation.literature import COLUMNS, CurationRelevance
+
+
+class TestTSV(unittest.TestCase):
+    """Tests for curated_papers tsv file."""
+
+    def setUp(self):
+        """Set up the test case."""
+        self.relevancy_types = {r.name for r in CurationRelevance}
+
+    def validate_row(self, row):
+        """Validate a single row from the TSV file."""
+        for field in COLUMNS:
+            self.assertIn(field, row)
+
+        self.assertTrue(row["pmid"].isdigit(), msg="PubMed identifier should be an integer")
+        self.assertTrue(row["pr_added"].isdigit(), msg="Pull Request should be an integer")
+
+        # Validate relevant is 0 or 1
+        self.assertIn(row["relevant"], ["0", "1"])
+
+        if row["relevant"] == "1":
+            prefix = row["prefix"]
+            self.assertIsNotNone(prefix, msg="prefix should be set for all relevant entries")
```

Comment:
I don't think this check is useful... again, my intention was to allow for relevant curations without having to have a perfect and complete assignment to an existing prefix. Let's revert this to not make curation really awkward.

Reply:
Alright, we can simplify this for now. In the future, it would be helpful to include some minimum context in PRs about the motivation and scope of experimental things like this. Alternatively, you can wait to send PRs for experimental projects, or keep them in separate repos to avoid the need for more detailed/stringent review that might be confusing for new contributors.

Reply:
Commented out the check for now, we could revisit formally annotating prefixes in the curation file in the future, if there is a use case for it.